beta-src/src/state/game/initial-state.ts

```diff
@@ -0,0 +1,98 @@
+import { GameState } from "../interfaces/GameState";
+
+const initialState: GameState = {
+  apiStatus: "idle",
+  error: null,
+  overview: {
+    anon: "Yes",
+    drawType: "draw-votes-public",
+    excusedMissedTurns: 4,
+    gameOver: "No",
+    members: [
```

Comment:
At this point, are we only focused on two player games?

Reply:
> At this point, are we only focused on two player games?

It will need to change when we get into the chat messaging development.  You can't create games with messaging with less than the full 7 players.

Reply:
i could just make it an empty array...this is the initial state...it doesn't mean that we are limited to 2 players.